### ISSUE TITLE
Liquid Phase example does not work

### ISSUE THREAD

**Issue by connie** [GreenGroup/RMG-Py#342]
Errored as follows:

```python
Generating thermodynamics for new species...
Found thermo for C in thermo library "corrected": primarythermolibrary
Error: Couldn't find in main abraham database:
Error: <Molecule "C">
Error: 1 C u0 p1 c0 {2,S} {3,S}
2 H u0 p0 c0 {1,S}
3 H u0 p0 c0 {1,S}

Traceback (most recent call last):
  File "../../../rmg.py", line 147, in <module>
    rmg.execute(args)
  File "/home/connie/code/RMG-Py/rmgpy/rmg/main.py", line 438, in execute
    self.initialize(args)
  File "/home/connie/code/RMG-Py/rmgpy/rmg/main.py", line 426, in initialize
    self.reactionModel.enlarge([spec for spec in self.initialSpecies if spec.reactive])
  File "/home/connie/code/RMG-Py/rmgpy/rmg/model.py", line 744, in enlarge
    spec.generateThermoData(database, quantumMechanics=self.quantumMechanics)
  File "/home/connie/code/RMG-Py/rmgpy/rmg/model.py", line 165, in generateThermoData
    return self.processThermoData(database, thermo0, thermoClass)
  File "/home/connie/code/RMG-Py/rmgpy/rmg/model.py", line 194, in processThermoData
    soluteData = database.solvation.getSoluteData(self)
  File "/home/connie/code/RMG-Py/rmgpy/data/solvation.py", line 602, in getSoluteData
    soluteData = self.getSoluteDataFromGroups(species)
  File "/home/connie/code/RMG-Py/rmgpy/data/solvation.py", line 662, in getSoluteDataFromGroups
    sdata = self.estimateSoluteViaGroupAdditivity(molecule)
  File "/home/connie/code/RMG-Py/rmgpy/data/solvation.py", line 757, in estimateSoluteViaGroupAdditivity
    self.__addGroupSoluteData(soluteData, self.groups['abraham'], molecule, {'*':atom})
  File "/home/connie/code/RMG-Py/rmgpy/data/solvation.py", line 791, in __addGroupSoluteData
    raise KeyError('Node has no parent with data in database.')
KeyError: 'Node has no parent with data in database.'
```

Update: duplicate of #325?

**Comment by connie** [comment]
I think the problem is that the node the molecule hits is one of the top ones: 

    Css
    1 * Css u0

  which has solute = None.  Then it searches the parents and finds that none of them have any solute data, producing KeyError and failing

        while node is not None and node.data is None:
            node = node.parent
        if node is None:
            print 'here2'
            raise KeyError('Node has no parent with data in database.')

I think we need to have a default value for the top node or something to prevent this from being an issue. Can we set the solutedata to be 0's for the top node? Or is there a different way to do it?

**Comment by bslakman** [comment]
I'm not sure where this CH2(S) is coming from... but in any case, setting all the solute data to be 0 will cause the thermo of that species to not be corrected for solvation. As it is now, all of the solute data values in the database is added to some 'intercept' value inside of ```rmgpy/data/solvation.py```. So if we want the top node values to truly be 0, we should make the values in the database the opposite of the intercept values.

**Comment by rwest** [comment]
I think probably better to assume CH2 is solvated like CH4 (than to assume it's not saturated at all), which is what used to happen, and probably what happens to CH2(T)  i.e. all radicals are saturated with H atoms before the solvation correction is estimated.
The problem is presumably that CH2(S) now has a lone pair and not two radicals, so is not being saturated the same way. If we didn't already have it in a themo database, how would this fare in the Benson group additive thermo? Would we have the same problem?

Once we know something more about how radicals behave in solution we can think about radical solvation estimates...

**Comment by connie** [comment]
The CH2(S) comes from loading the primaryThermoLibrary.  

I think in thermo benson groups it would fall to the top group R, and that I believe has no corrections.  What is missing from the solvation is complete lack of data in the top group.  I think we should fill it in so that these kind of errors stop occurring.  Or at least some warning + result.

**Comment by rwest** [comment]
I think the conclusion of discussion with @whgreen was to treat it as we would a triplet diradical, i.e. put two H atoms on it before searching the Platts groups to find Abraham parameters. This could be applied to all lone pairs that we don't know how to treat (is that all lone pairs? maybe not... how can we tell which are problems?)
One thing that came up in discussion was that we should remove the H-bond-donating contribution of any H atoms that we add during this process. This should also be happening for radicals. After discussing with @bslakman, I don't think this is yet implemented in RMG-Py. This is where it was implemented in RMG-Java: https://github.com/GreenGroup/RMG-Java/commit/2c630c9fda301e5b43057f247bda1e55eab87def

**Comment by bslakman** [comment]
Closed via efb4d1d
